frontend-ssr/pages/submissions.tsx

```diff
@@ -36,6 +42,27 @@ class Submissions extends React.Component<ConnectedProps<typeof connector>, Stat
     if (submissions.submissions === undefined) {
```

Comment:
Now that we have the intersection observer logic, we can get rid of this code:

```
  let { submissions } = this.props;
    if (submissions.submissions === undefined) {
      this.getData();
    }
```

and just fetch based on the intersection, because  when you enter the page -  if there are NO elements it will be intersecting and start the fetch. Otherwise (if it's a return visit) - there will be elements and no fetch will happen.

For this to work you'll also need to revert 

```
this.state = { loading: false }
```